### DISCUSSION TITLE
@Html.Emebeded("imagepath") has error

### DISCUSSION THREAD

**Discussion by CrucialSoftware** [community/community#49695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,
I have tried Postman for .net core, it is working fine. Thank you for that.
But when I am using @Html.Embeded for Image, it is throwing error  for null reference.
Can I have any idea to resolve it?

Thank you.

**Comment by CrucialSoftware** [comment]
Hello,
I tried Embedded also it shows  'does not contain definition ..' error, I
have also applied @using Postman

[image: image.png]


On Fri, Mar 10, 2023 at 6:08 PM Mohamed Sayed Al-Ahl <
***@***.***> wrote:

> It looks like there might be a typo in your code - instead of
> ***@***.***", it should be ***@***.***". Try correcting the typo
> and see if that resolves the issue. Additionally, make sure that the
> "imagepath" variable is not null and that the image file exists in the
> specified path. If the issue persists, please provide more details about
> the error message you are receiving.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/49695#discussioncomment-5267030>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHB4GKETN3XYEAYV6WYKVITW3MODLANCNFSM6AAAAAAVWNAVFM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Best Regards, Siddharajsinh Jadeja*

*Director *
Crucial Software and IT Park Pvt. Ltd.

*Office Address:*
D-513, VTC,
Nr. Reliance Circle,
Kudasan,
Gandhinagar, GUJ-India
Zip : 382421

*(M) +91 8469922322*



(w) www.csitpark.com
(fb) Like Us on Facebook <https://www.facebook.com/csitpark>
(g+) Google+ <https://plus.google.com/+CSITPark>

*Skype*: csitpark

*Kindly return the copy of this original message when replying. Thank you.*

**Comment by otter-computer** [comment]
Welcome to the GitHub Community, @CrucialSoftware, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
